docs/runtime/runtime-metadata.md

```diff
@@ -0,0 +1,196 @@
+                    "type":{
+                        "PlainType":"AccountId"
+                    },
+                    "fallback":"0x0000000000000000000000000000000000000000000000000000000000000000",
+                    "documentation":[
+                        " The `AccountId` of the sudo key."
+                    ]
+                }
+            ],
+            "calls":[
+                {
+                    "name":"sudo",
+                    "args":[
+                        {
+                            "name":"proposal",
+                            "type":"Proposal"
+                        }
+                    ],
+                    "documentation":[
+                        " Authenticates the sudo key and dispatches a function call with `Root` origin.",
+                        "",
+                        " The dispatch origin for this call must be _Signed_."
+                    ]
+                },
+                {
+                    "name":"set_key",
+                    "args":[
+                        {
+                            "name":"new",
+                            "type":"Address"
+                        }
+                    ],
+                    "documentation":[
+                        " Authenticates the current sudo key and sets the given AccountId (`new`) as the new sudo key.",
+                        "",
+                        " The dispatch origin for this call must be _Signed_."
+                    ]
+                }
+            ],
+            "events":[
+                {
+                    "name":"Sudid",
+                    "args":[
+                        "bool"
+                    ],
+                    "documentation":[
+                        " A sudo just took place."
+                    ]
+                },
+                {
+                    "name":"KeyChanged",
+                    "args":[
+                        "AccountId"
+                    ],
+                    "documentation":[
+                        " The sudoer just switched identity; the old key is supplied."
+                    ]
+                }
+            ]
+        }
+    ]
+}
+```
+
+If you are using other languages to build your client, you can easily send **WebSocket message** or **HTTP POST request** to Substrate node endpoint with any existing client. The message or body for `getMetadata` is:
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "state_getMetadata",
+  "params": []
+}
+```
+
+The response looks like following content, but with much more information in `result` field:
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": "0x6d6574610324187379737......44964244163636f756e74496400",
+    "id": 1
+}
+```
+
+The hexadecimal string in `result` field conforms to SCALE data format. Unfortunately there are no existing tools for languages except Rust and Javascript, which help you encode your request data and decode the response blob.
```

Comment:
There are more implementations than that. There's https://github.com/ChainSafe/gossamer and https://github.com/opennetsys/golkadot/ in **Go**, https://github.com/soramitsu/kagome in **C++** , https://github.com/polkascan/py-scale-codec / https://github.com/polkascan/polkascan-pre-explorer-api  in **Python** and one in **MySQL** (which I don't have the link to)

Reply:
Listed as the "Polkadot Implementations" here https://forum.web3.foundation/t/teams-building-on-polkadot/67

Reply:
Thanks for the correction and detail explanation.
I also raised [an issue](https://github.com/paritytech/parity-scale-codec/issues/133) to put information about more implementations in `parity-scale-codec`.